scripts/packagerScript.js

```diff
@@ -8,9 +8,8 @@ const { notarize } = require('electron-notarize');
 
 async function notarizing(context) {
   const { electronPlatformName, appOutDir } = context;
-  if (electronPlatformName !== 'darwin') {
-    return;
-  }
+  if (electronPlatformName !== 'darwin') return;
```

Comment:
why add another if check instead of expanding existing one?

Reply:
Fixed